src/shape.rs

```diff
@@ -278,6 +278,12 @@ impl Shape {
         offset_indent.to_string_inner(config, 0)
     }
 
+    pub(crate) fn to_string(&self, config: &Config) -> Cow<'static, str> {
```

Comment:
Probably worth adding a doc comment so that we know that this function gives us a single line of indentation.